.github/ISSUE_TEMPLATE/new-module-proposal.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New module proposal
+about: Propose a new module for the Performance Lab plugin
+title: 'Module Proposal: '
+labels: Needs Discussion, no milestone, [Type] Module
```

Comment:
I don't think we need the `no milestone` label on these. A bit unsure about `Needs Discussion` as well, maybe we should limit the defaults to just the label that this needs for sure (which is `[Type] Module`)?

Also I'm not sure the formatting here is correct, according to https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms this needs to be an array I think?
```suggestion
labels: [ "[Type] Module" ]
```

Reply:
@felixarntz I didn't do anything with formatting, just did this in the UI, so again, not sure how to change this. I'll have to set up a local and try updating the actual files tomorrow instead and I can update the labels then, too.